### ISSUE TITLE
inline param is sometimes highlighted

### ISSUE THREAD

**Issue by bishabosha** [scala/vscode-scala-syntax#148]
```scala
inline def toIntMacro(inline nat: Nat): Int = ${ Macros.toIntImpl('nat) } // `(inline nat: Nat)` - inline is not highlighted

inline def toIntUnapply(inline nat: Nat): Int = inline 1 match // `(inline nat: Nat)` - inline is highlighted
```

**Comment by nicolasstucki** [comment]
This was already fixed with #150

**Comment by nicolasstucki** [comment]
Should add a regression test with `match`
